recipes/u-boot/u-boot-tools.inc

```diff
@@ -44,9 +44,7 @@ inherit auto-package-utils
 U_BOOT_TOOLS = "bmp_logo mkimage envcrc gen_eth_addr img2srec img2brec.sh \
     jtagconsole netconsole setlocalversion"
 
-GDB_TOOLS:machine = ""
```

Comment:
The install snippet in +58 checks if the tools are built, so no need to do {machine,sdk,native} handling anymore.